L1Trigger/Phase2L1ParticleFlow/interface/conifer.h

```diff
@@ -0,0 +1,159 @@
+#ifndef CONIFER_CPP_H__
+#define CONIFER_CPP_H__
+#include "FWCore/Utilities/interface/Exception.h"
+#include "nlohmann/json.hpp"
```

Comment:
is this library packaged with standard CMSSW?

Reply:
It's an external, for example at `/cvmfs/cms.cern.ch/slc7_amd64_gcc900/external/json/3.10.2-8cd3895745e2546d082bb9980a719047/include`